lib/rubocop/cop/minitest/empty_line_before_assertion_methods.rb

```diff
@@ -40,12 +40,15 @@ def on_send(node)
 
         private
 
+        # rubocop:disable Metrics/CyclomaticComplexity
         def assertion_method(node)
           return node if assertion_method?(node)
           return unless (parent = node.parent)
+          return if parent.block_type? && parent.method?(:test)
 
           node.parent if parent.block_type? && parent.body && assertion_method?(parent.body)
```

Comment:
Can you update as below? `rubocop:disable` and `rubocop:enable` comments can be removed.

```suggestion
          return unless parent.block_type? 
          return if parent.method?(:test)

          node.parent if parent.body && assertion_method?(parent.body)
```